Formula/ydcv.rb

```diff
@@ -0,0 +1,17 @@
+class Ydcv < Formula
+  desc "YouDao Console Version - Simple wrapper for Youdao online translate (Chinese <-> English) service API, as an alternative to the StarDict Console Version(sdcv)."
```

Comment:
Description should be 70 characters maximum

Reply:
Done